coins/ethereum/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "ethereum"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+thiserror = "1"
+serde_json = "1.0"
+serde = "1.0"
+hex-literal = "0.3"
+ethers = { git = "https://github.com/gakonst/ethers-rs", features = ["abigen", "ethers-solc"] }
+eyre = "0.6"
+tokio = { version = "1.19", features = ["macros"] }
+k256 = { version = "0.11", features = ["arithmetic", "keccak256", "ecdsa"] }
+
+serai-processor = { path = "../../processor" }
+frost = { package = "modular-frost", path = "../../crypto/frost", features = ["secp256k1", "ed25519"] }
```

Comment:
ed25519 shouldn't be a needed feature. serai-processor should import ethereum, not the other way around. I did debate if this should solely be a processor integration? Because we're not building a full code lib? But between the Schnorr support, which is generic over multisig, and the discussed inner call data retrieval... there's reason to make it a lib.

Reply:
updated